docs/VR/ChangeRegistration/index.md

```diff
@@ -13,11 +13,11 @@ Purpose
 
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
-|vrSynonyms|String|false|array: true<br>minsize: 1<br>maxsize: 100<br>maxlength: 40|
-|language|[Common.Language](../../common/enums/#language)|true||
-|appID|Integer|true||
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:---------|
+|vrSynonyms|String|false|array: true<br>minsize: 1<br>maxsize: 100<br>maxlength: 40|Request new VR synonyms registration<br>Defines an additional voice recognition command.<br>Must not interfere with any name of previously registered applications from the same device.|
```

Comment:
The new description 
```
Must not interfere with any name of previously registered applications from the same device.
```
should also be added to the `vrSynonyms` param in the `HMIApplication` struct and the `BasicCommunication.OnAppRegistered` RPC

Reply:
added description with 10ecd74